test/FastTests/Voron/Tables/DynamicBTreeIndex.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System.Text;
+using Sparrow;
+using Sparrow.Server;
+using Voron;
+using Voron.Data.Tables;
+using Voron.Util.Conversion;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Voron.Tables
+{
+    public unsafe class DynamicBTreeIndex : TableStorageTest
```

Comment:
I don't like that the name of the test is the same as newly introduced type. Just use issue number?